src/pages/login/LoginPage.js

```diff
@@ -60,10 +60,12 @@ function LoginPage() {
           type="password"
           name="password"
           id="password"
-        required
+          required
         ></input>
 
-        <button type="submit">Submit</button>
+        <Button className="submit-button" type="submit">
```

Comment:
this `type="submit"` won't work how you have it. Right now it's being passed as a prop, which you are not receiving in the `<Button />` component so it's not actually being applied to the `<button>` element in the JSX of the Button component.

If you want to be able to determine the type of button you will need to pass type as a prop (which you are doing here!) AND make sure you are taking that prop in in the `<Button /> `component.

Reply:
Thats right I never added it as props. It's updated now!